package-managers/dnf-harden.conf

```diff
@@ -0,0 +1,4 @@
+[main]
+deltarpm=0
+zchunk=0
+gpgcheck=1
```

Comment:
While the other two are okay, this one may break stuff. Specifically, it will definitely break repos without `gpgkey` defined, but also may break some where it was intentionally disabled (by commenting `gpgcheck=1` out instead of setting it to 0).  And when it breaks, it will interrupt the whole transaction, basically breaking all updates (among other things, making it impossible to get fix for this issue without manual intervention). This may also break installing packages from files (like locally built ones), not sure.
I know _you_ probably would never use such configuration, but some users may want to. All repositories definitions that are supposed to have signatures checked already have `gpgcheck=1`.